templates/registration/registration_form.html

```diff
@@ -79,8 +79,9 @@
         #edit-form {
             border: unset;
             background: unset;
-            max-width: 450px;
+            max-width: 100%;
             width: 450px;
+            margin: 0 -10px 0 -15px;
```

Comment:
If you set `#edit-form`'s max-width to 90% and `#center-float`'s `max-width` to 100%, you don't need this margin hack.

Reply:
It makes the form not fully centering on large screens (i.e laptop)